setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name            = "solus-sc",
-    version         = "15.0",
+    version         = "14.0",
```

Comment:
Problem is that now your pull request changes the version again, this time Ikey did bump it to 15.0 in e8ac37dcf7511990125b7f964e474d5e593dc5d1